src/Symfony/Component/JsonEncoder/DependencyInjection/EncodablePass.php

```diff
@@ -30,7 +30,11 @@ public function process(ContainerBuilder $container): void
         $encodableClassNames = [];
 
         // retrieve concrete services tagged with "json_encoder.encodable" tag
-        foreach ($container->findTaggedServiceIds('json_encoder.encodable') as $id => $tags) {
```

Comment:
Is there an issue with `findTaggedServiceIds`? This is the right method to use.

Reply:
When I have registered the `JsonEncodable` attribute for autoconfiguration, I added the `container.excluded` tag so that it isn't really considered as a service (see https://github.com/symfony/symfony/pull/59401/files#diff-e4f52d5033296421d02e360c8355aa1931bdfb0c12579f4815ea02b724322c0cR745-R748). And, the `findTaggedServicesIds` ignores the `container.excluded` services, so I used the "by-hand way".

But I can remove the `container.excluded` tag and use again the `findTaggedServiceIds` if you prefer.